core/src/apps/solana/transaction/instruction.py

```diff
@@ -0,0 +1,145 @@
+from typing import TYPE_CHECKING
+
+from trezor.utils import BufferReader
+from trezor.wire import ProcessError
+
+from .parse import parse_property
```

Comment:
This import and the two above seems to be used only in one function - then it makes sense to import it locally in that function.